inst/www/shared/shiny.js

```diff
@@ -3296,6 +3299,14 @@
         sendOutputHiddenStateDebouncer.immediateCall();
     });
 
+    function filterEventsByNamespace(namespace, handler) {
+      return function(e) {
+        if (e.namespace === namespace) {
```

Comment:
Does this exact comparison do the right thing? I would have expected a substring match to be needed, splitting on `.`. I have a little test in a  jsfiddle here: http://jsfiddle.net/winstonchang/b4n8ccyc/